### ISSUE TITLE
"Pin" Sticker Sheet in new window

### ISSUE THREAD

**Issue by txcrew** [romannurik/Sketch-Stickers#18]
Hi there,

I would like to see the addition of a feature which would allow the designer to keep the sticker sheet open in a separate window (i.e. a 2nd monitor of stickers), rather than having it overlay Sketch every time.

Thanks for the great plugin!

**Comment by romannurik** [comment]
Yes good idea! I started building this mode already a while ago but haven't
worked on it enough to publish it.

**Comment by romannurik** [comment]
FYI I don't have any plans to further develop this mode unless a lot of folks really want it. For now, anyone can play with it by setting `UI_MODE` to `palette` in [stickers-ui.js](https://github.com/romannurik/Sketch-Stickers/blob/master/src/stickers-ui.js#L28) and rebuilding the plugin locally.
